src/main/kotlin/uk/gov/justice/digital/needs/HmppsNeedsApiApplication.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.needs
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class HmppsNeedsApiApplication
+
+fun main(args: Array<String>) {
+    runApplication<HmppsNeedsApiApplication>(*args)
+}
```

Comment:
Do we need this at all? Isn't the existing main ok?

I'm a bit frightened that having two classes annotated with @SpringBootApplication is going to behave is a spooky way.

Reply:
Same with the jpa and OpenApi config.

Reply:
Taken out the duplicate files :)